apprentice_teacher.rb

```diff
@@ -1,21 +1,36 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
-
-  def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
-    @target_raise = 800
-    @phase = 3
-  end
+require_relative 'teacher'
+require_relative 'praise'
 
-  def offer_high_five
-    "High five!"
-  end
+class ApprenticeTeacher < Teacher
+  
+  include Praisable
+  # Here i want to only take from the super class these attributes.
+  # But I don't know how to do that
+ 
+  # super( attr_reader :age, :salary, :phase, :target_raise)
```

Comment:
You can only call super from within a method, and then you pass through whatever args you want to send to the version of that same method in the superclass.  As it is, you're calling it outside the context of a method and passing a method call to `attr_reader` (which generates methods for your class).